nlp/admin.js

```diff
@@ -57,9 +57,8 @@ router.post('/reload/@:model_tag/:locale', async (req, res, next) => {
         return;
     }
 
-    model.reload().then(() => {
-        res.json({ result: 'ok' });
-    }).catch(next);
+    await model.reload();
+    res.json({ result: 'ok' });
```

Comment:
Not sure if this is the right fix.  model.reload() doesn't return a promise.
Similar issue was found in 1.16.1 where reload() return null.

Reply:
`model.reload()` should return a promise, see #464